packages/analytics/src/Analytics.ts

```diff
@@ -236,10 +236,10 @@ class Analytics {
    *
    * @returns Promise that will resolve when the method finishes.
    */
-  protected async onLoadedIntegrations(
+  protected onLoadedIntegrations(
```

Comment:
We need to keep the flexibility of returning a promise for overriders of this method. To do so and be type-compatible with the override in AnalyticsWeb you can change this signature to this:

```suggestion
  protected onLoadedIntegrations(
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
    loadedIntegrations: IntegrationRuntimeData[],
    ): Promise<void> | void {....
```